package.json

```diff
@@ -14,7 +14,8 @@
     "format:ios": "find apple/ -iname *.h -o -iname *.m -o -iname *.cpp -o -iname *.mm | xargs clang-format -i",
     "lint:js": "eslint --ext '.js,.ts,.tsx' src/ example/src FabricExample/src MacOSExample/src && yarn prettier --check './{src,example,FabricExample,MacOSExample}/**/*.{js,jsx,ts,tsx}'",
     "lint:js-root": "eslint --ext '.js,.ts,.tsx' src/ && yarn prettier --check './src/**/*.{js,jsx,ts,tsx}'",
-    "lint:android": "./android/gradlew -p android spotlessCheck -q"
+    "lint:android": "./android/gradlew -p android spotlessCheck -q",
+    "checkIntegrity": "(cd ./android && ./gradlew checkIntegrityBetweenArchitectures)"
```

Comment:
`checkIntegrity` is very generic, can we use something more specific to checking codegen artifacts?